src/1-basic/06-set-properties.js

```diff
@@ -5,27 +5,27 @@ class CustomInput extends LitElement {
 
   static get properties() {
     return {
-      helpText: String,
-      placeholder: String,
+      helpText: { type: String },
+      placeholder: { type: String },
     };
   }
 
-  _render({ helpText, placeholder }) {
+  render() {
     return html`
       <div>
         <div>
-          <input placeholder$="${placeholder}">
+          <input .placeholder="${this.placeholder}">
```

Comment:
`<input placeholder="${this.placeholder}">` should do

Reply:
Thanks! It is enough indeed.